<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeroSkillsWidget</class>
 <widget class="QDialog" name="HeroSkillsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hero skills</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="statsLayout">
     <item>
      <widget class="QLabel" name="labelAttack">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="attack"/>
     </item>
     <item>
      <widget class="QLabel" name="labelDefence">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="defence"/>
     </item>
     <item>
      <widget class="QLabel" name="labelPower">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="power"/>
     </item>
     <item>
      <widget class="QLabel" name="labelKnowledge">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="knowledge"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="skills">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>120</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <column>
      <property name="text">
       <string>Skill</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Level</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Customize skills</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
